packages/vue-components/src/Tooltip.vue

```diff
@@ -71,9 +71,9 @@ export default {
 </script>
 
 <style>
-    .v-popper--theme-tooltip .v-popper__inner {
+    .v-popper-theme-tooltip .v-popper-inner {
```

Comment:
This seems to be having unexpected effects - the styling doesn't apply to the tooltip anymore. 
You can try modifying the font size to something huge and then removing the one of the dashes, you realise the font change doesn't apply. 
Can we keep the old version? Will it automatically change every time the linter is run?

Reply:
<img width="835" alt="Screenshot 2024-02-28 at 9 19 24 PM" src="https://github.com/MarkBind/markbind/assets/122192553/67ac00e6-abff-44af-b6fa-24ebbb8c4d6f">

The namings here violated the rule `selector-class-pattern`. 
To revert it back to the old version and prevent stylelint from throwing an error, I have added a line comment on top of it to disable stylelint for this part. (this is how stylelint is disabled for a particular part of code)